domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/mapper/FieldTypeListItemMapper.java

```diff
@@ -4,26 +4,43 @@
 import uk.gov.hmcts.ccd.definition.store.repository.model.FixedListItem;
 
 import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class FieldTypeListItemMapper {
     public List<FixedListItem> entityToModel(final List<FieldTypeListItemEntity>
```

Comment:
I'm not 100% sure a Mapper is the right place to do the sorting based on Display order. Probably the mapper should just do 1 to on mapping and nothing more than that.